dbms/src/main/java/org/polypheny/db/PolyphenyDb.java

```diff
@@ -261,9 +260,6 @@ public void join( final long millis ) throws InterruptedException {
             throw new RuntimeException( "Something went wrong while initializing index manager.", e );
         }
 
-        // Call DockerManager once to remove old containers
-        DockerManager.getInstance();
-
```

Comment:
Why removing this call?

Reply:
@cedricmendelin  maybe this  did happen during merge?
I am not aware that I removed it.

But I fixed it now.